content/en/docs/setup/install/halyard.md

```diff
@@ -64,6 +64,10 @@ Reference](/docs/reference/halyard/commands).
 
 ### Update Halyard on Debian/Ubuntu
 
+> __Note__: This upgrade procedure only works with Halyard v1.45.0 and higher.
+If you are using a Halyard version prior to v1.45.0, you should follow the
```

Comment:
```suggestion
If you are using a Halyard version prior to v1.45.0, follow the
```

Reply:
@dbyron-sf Thanks for the review, I've made the change as suggested 👍